cmd/create/cluster/cmd.go

```diff
@@ -136,37 +135,33 @@ func init() {
 		"",
 		"The Amazon Resource Name of the role that OpenShift Cluster Manager will assume to create the cluster.",
 	)
-	flags.MarkHidden("role-arn")
 	flags.StringVar(
 		&args.externalID,
 		"external-id",
 		"",
-		"A unique identifier that might be required when you assume a role in another account",
+		"A unique identifier that might be required when you assume a role in another account.",
```

Comment:
This might be better to facilitate the help in interactive mode since we don't explicitly say its "optional" `Optional identifier information when assuming a IAM role`

Reply:
Fixed